File: src/test/java/net/dinkla/raytracer/samplers/MultiJitteredTest.java
package net.dinkla.raytracer.samplers;

import org.junit.jupiter.api.BeforeEach;

public class MultiJitteredTest extends GeneratorTest {

    @BeforeEach
    @Override
    public void initialize() {
        new MultiJittered().generateSamples(NUM_SAMPLES, NUM_SETS , samples);
    }

}

File: src/test/kotlin/net/dinkla/raytracer/samplers/MultiJitteredTest.kt
package net.dinkla.raytracer.samplers

import net.dinkla.raytracer.math.Point2D

class MultiJitteredTest : AbstractGeneratorTest() {

    override val sizeOfXY: Int = 100
    override val numberOfSamples: Int = NUM_SETS * IGenerator.sqrt(NUM_SAMPLES) * IGenerator.sqrt(NUM_SAMPLES)  + 1

    override fun sample(): MutableList<Point2D> =
            MultiJittered.generateSamples(NUM_SAMPLES, NUM_SETS)

}
